packages/rspack-test-tools/tests/configCases/builtins/define/index.js

```diff
@@ -144,25 +144,12 @@ it("should builtins define works", () => {
 	} catch (err) {}
 	expect(error_count).toBe(6);
 
-	try {
-		error_count += 1;
-		SHOULD_CONVERTED = 205; // syntax error
-		error_count += 1;
-	} catch (err) {}
-	expect(error_count).toBe(7);
-
-	try {
-		error_count += 1;
-		SHOULD_CONVERTED = SHOULD_CONVERTED = 205; // syntax error
```

Comment:
not supported by webpack, now it's a SyntaxError in rspack (assigning literal to literal)